### DISCUSSION TITLE
Error Definition for rule x was not found

### DISCUSSION THREAD

**Discussion by brayanarrieta** [testing-library/eslint-plugin-testing-library#864] [General]
I'm receiving the next error with the library once I defined the rules and execute the eslint command. I received an error for every rule used. My setup is the same as the Readme mentioned

```
 error  Definition for rule 'testing-library/await-async-queries' was not found
```

**Comment by Belco90** [comment]
Hi @brayanarrieta. I'm afraid we need more details. Did you install the plugin? Did you load the plugin correctly in your ESLint config file? Sharing the setup or a small demo would be nice.

**Reply by brayanarrieta** [reply]
Issue solved was related to a wrong version installed after debug the issue was fixed

Thx @Belco90 for the response quick
